app/models/bar.py

```diff
@@ -136,7 +138,7 @@ class BarDailyData(db.Model):
 
     balance_change: Column[float] = column(sa.Numeric(6, 2, asdecimal=False), nullable=False, default=0.0)
     items_bought_count: Column[int] = column(sa.Integer(), nullable=False, default=0)
-    alcohol_bought_count: Column[int] = column(sa.Integer(), nullable=False, default=0)
+    alcohol_bought_count: Column[int] = column(sa.Numeric(4, 2, asdecimal=False), nullable=False, default=0.0)
```

Comment:
Du coup il faut transitionner toutes les data existantes dans la migration

```suggestion
    alcohol_bought_count: Column[float] = column(sa.Numeric(4, 2, asdecimal=False), nullable=False, default=0.0)
```